src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -230,6 +233,37 @@ private boolean isDamageFromBombingDoneToUnitsInsteadOfTerritories(final GameDat
 
   public static void setUnitFlagDrawMode(final UnitFlagDrawMode unitFlag, final Preferences prefs) {
     drawUnitNationMode = unitFlag;
-    prefs.put("UNIT_FLAG_DRAW_MODE", unitFlag.toString());
+    prefs.put(UNIT_FLAG_DRAW_MODE, unitFlag.toString());
+  }
+  
+  private static boolean blockInputs = false;
+  
+  public static KeyListener getFlagToggleKeyListener(TripleAFrame frame) {
```

Comment:
Why is this declared here, but only used in one other place, from a different class? Can we move it to the class that uses it?